frontend/pages/group/data/foods.vue

```diff
@@ -326,8 +331,21 @@ export default defineComponent({
         value: "onHand",
         show: true,
       },
+      {
+        text: i18n.tc("general.date-added"),
+        value: "createdAt",
+        show: false,
+      }
     ];
 
+    function formatDate(date: string) {
+      try {
+        return i18n.d(Date.parse(date), "medium");
```

Comment:
I'm going to leave this only because it uses i18n so it's translated. We could modify it to only display yyyy-mm-dd but as you said I'm not sure how much it matters since it's really just for sorting anyway

Reply:
Yeah, that sounds more important than removing the weekday